Shift3/boilerplate-client-react
# IPS1A - Initialized project with CRA

## Changes
Add Create React App to initial project setup by running:
 [ `npx create-react-app client --template typescript`] from project

## Purpose
Currently there was no project initialized. This is a custom template from Create React App that also adds Typescript starter files. For more information visit https://create-react-app.dev/docs/custom-templates

## Approach
Researched tools for React. 

## Learning
After researching other tooling such as webpack, the conclusion to use Create React App with the Template Typescript seemed to fit the needs of the project, while making the CLI more manageable over time due to its built in updates.

## Testing
1. Pull in the changes to your local copy of this branch and run Yarn Start.
2. Make sure project spins up.

## Screenshots

### Closes #52
